meta_merkle_tree/src/meta_merkle_tree.rs

```diff
@@ -210,6 +210,12 @@ mod tests {
         signature::{EncodableKey, Keypair},
```

Comment:
does `new_test_key` here need to write to a file still?

Reply:
Good point - no it doesn't, refactoring to just returning a new keypair pubkey.

Reply:
`test_write_merkle_distributor_to_file` - should I just comment out this test for us as well? Jito had this code in here beofrehand but it really doesn't have any application for what we are doing..